fsspec/implementations/zip.py

```diff
@@ -55,6 +56,9 @@ def __init__(
         self.block_size = block_size
         self.dir_cache = None
 
+        if kwargs.get("_info_implementation") == "base":
```

Comment:
The plan is to remove this, as soon as the case is made for the new functionality, right?